### DISCUSSION TITLE
Error services.vwsfriend.depends_on contains an invalid type, it should be an array

### DISCUSSION THREAD

**Discussion by hofmarkus** [tillsteinbach/VWsFriend#144] [Q&A] [answered]
I have installed vwfriend and configure out myconfig.env, but when I would start the docker I see this error:

docker-compose --env-file ./myconfig.env up
ERROR: The Compose file './docker-compose.yml' is invalid because:
services.vwsfriend.depends_on contains an invalid type, it should be an array

I have download the version 0.12.0 and the latest, but the same result.
Please could you help me to resolve the problem. 
Thanks
 
docker version
Client: Docker Engine - Community
 Version:           20.10.12
 API version:       1.41
 Go version:        go1.16.12
 Git commit:        e91ed57
 Built:             Mon Dec 13 11:45:33 2021
 OS/Arch:           linux/amd64
 Context:           default
 Experimental:      true

Server: Docker Engine - Community
 Engine:
  Version:          20.10.12
  API version:      1.41 (minimum version 1.12)
  Go version:       go1.16.12
  Git commit:       459d0df
  Built:            Mon Dec 13 11:43:42 2021
  OS/Arch:          linux/amd64
  Experimental:     false
 containerd:
  Version:          1.4.12
  GitCommit:        7b11cfaabd73bb80907dd23182b9347b4245eb5d
 runc:
  Version:          1.0.2
  GitCommit:        v1.0.2-0-g52b36a2
 docker-init:
  Version:          0.19.0

docker-compose --version
docker-compose version 1.26.2, build eefe0d31

**Comment by tillsteinbach** [comment, marked as answer]
Nothing my requirements are outdated. Could you check if it works after upgrading to at least docker-compose 1.27.0?

**Reply by hofmarkus** [reply]
With the new docker-compose version it now works without problems. (Docker Compose version v2.2.3)
Thank for your help.
